Example/jaegerMediator/app.js

```diff
@@ -23,7 +23,7 @@ app.get('/sampleSpan', (req, res) => {
 
 // API: Report spans to a Jaeger Collector
 app.post('/spans', function (req, res) {
```

Comment:
I guess the endpoint can now be called `/batch`

Reply:
yes!